README.md

```diff
@@ -74,7 +74,38 @@ hubspot.contacts.get(function(err, results) {
   console.log(results);
 });
 ```
-
+## {EXAMPLE} Create Contact
+```javascript
+const contactObject = {
+            "properties":
+            [
+                { "property": "firstname","value": yourvalue },
+                { "property": "lastname", "value": yourvalue },
+                { "property": "email", "value": yourvalue },
+                { "property": "phone", "value": yourvalue },
+                { "property": "state", "value": yourvalue },
+                { "property": "non_us", "value": yourvalue },
+                { "property": "company", "value": yourvalue },
+                { "property": "hubspot_owner_id", "value": yourvalue },
+                { "property": "lifecyclestage", "value": yourvalue },
+                { "property": "hs_lead_status", "value": yourvalue },
+                { "property": "type", "value": yourvalue },
+                { "property": "source", "value": yourvalue },
+                { "property": "jobtitle", "value": yourvalue },
+                { "property": "address", "value": yourvalue },
+                { "property": "city", "value": yourvalue }
+            ]
+        };
+
+          const hubspot = new Hubspot({ apiKey: YOUR API KEY });
+          const hubspotContact = await hubspot.contacts.create(contactObj);
+```
+## {EXAMPLE} If you need to insert mulitple values. each value must have a semi colon after each value
+```javascript
+const arrayOfValues = ["value1", "value2", "value3", "value4"];
```

Comment:
I'm not sure I understand this. Do you need to use this for a specific HubSpot data type or is it a general explanation (to cast an array into a string value)?

Reply:
Yes if you send hubspot multiple items it must be delimited by a semi colon.